cognite_toolkit/_cdf_tk/commands/_profile.py

```diff
@@ -356,13 +360,17 @@ class Columns:
         ColumnCount = "Columns"
 
     is_dynamic_table = True
-    profile_row_limit = 10_000  # The number of rows to profile to get the number of columns.
+    max_profile_row_limit = 10_000  # The number of rows to profile to get the number of columns.
```

Comment:
note that profiler-api internal limit is 256MB of RAW data https://github.com/cognitedata/profiler-api/blob/main/src/main/scala/com/cognite/raw_profiler/Profile.scala#L37 and you can omit limit= parameter to get max (if it fits the request timeout..)

On exact threshold don't know what'd work best for toolkit, leaving a note just in case you'll want to experiment on profiler<>transformation preview threshold